requirements.txt

```diff
@@ -2,10 +2,15 @@ alembic==0.9.5
 -e git+https://github.com/elifesciences/bus-sdk-python.git@49a0b0ecc55f99bc55a61f3e9b613ea88e3ff5a5#egg=bus-sdk-python
 -e git+https://github.com/elifesciences/api-validator-python.git@2971a2146321d48cc88c9c9f36e8fd72740dbfee#egg=api-validator-python
 astroid==1.5.3
+attrs==17.3.0
 blinker==1.4
+boto3==1.4.7
+botocore==1.7.36
```

Comment:
No `pip freeze` recently? (Maybe CI should check?)

Reply:
that `pip freeze`'s output is identical to this file?

Reply:
Yeah, so it's always up to date.